src/acf.py

```diff
@@ -185,6 +208,12 @@ def ros_setup(self):
         self.create_service(SetFloat, '/ACF/set_payload', self.set_payload)
         self.create_service(SetFloat, '/ACF/set_f_zero', self.set_f_zero)
         self.create_service(SetDuration, '/ACF/set_t_ramp', self.set_t_ramp)
+        if self.frequency > 0:
+            self.create_timer(1 / self.frequency, self.timer_callback)
+        if self.joint_name != "":
```

Comment:
```suggestion
        if self.joint_name:
```

is more *pythonic* (empty strings automatically evaluate to `False` or `None`), but perhaps doesn't show intent as well as being explicit in the comparison.

Reply:
In this case I prefer being explicit here, since the parameter default is also explicitly set to a blank string it makes the intent more clear.